contribs/gmf/less/vars.less

```diff
@@ -30,3 +30,4 @@
 
 @brand-primary: #b1c2b5;
 @brand-secondary: #d3e5d7;
+@font-size-small: ceil(@font-size-base * 0.9);
```

Comment:
Bootstrap 4 will use rem, than a ceil or a floor will do bad things (e.g. round on 10 pixels...)

See « Dropped IE8 support and moved to rem and em units. » on http://blog.getbootstrap.com/2015/08/19/bootstrap-4-alpha/